e2e/pages/app/toolbar-search/toolbar-search.testpage.component.ts

```diff
@@ -30,4 +31,38 @@ export class ToolbarSearchTestPageComponent {
         // Execute search here
         this.searchedFor = this.form.controls.search.value;
     }
+
+    @ViewChild('searchField')
```

Comment:
A lot of the variables and functions in this class are no used anywhere and should be removed. Also class properties should be above any methods. I also notice a few methods are missing return types.